### PR TITLE
Feature/jesse nitro

### PR DESCRIPTION
**Pull request by jschoenemann-cms**
Added tag manager

### CODE REVIEW

File: test/Components/GoogleTagManagerHeader.spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import GoogleTagManagerHeader from "~/components/gtm/GoogleTagManagerHeader"
+
+describe("GoogleTagManagerHeader", () => {
+  beforeAll(async () => {
+    await beforeAllTests()
+  })
+
+  test("if it has a tag key init tag manager", () => {
+    config.mocks.$config = {
+      tagManagerKey: "daKey",
```
Comment (fpigeonjr) [inline test/Components/GoogleTagManagerHeader.spec.js:12]: I approve

Comment (fpigeonjr) [inline test/Components/GoogleTagManagerHeader.spec.js:12]: I approve

Comment (jschoenemann-cms) [reply test/Components/GoogleTagManagerHeader.spec.js:12]: Should I make it daBears?

Comment (fpigeonjr) [reply test/Components/GoogleTagManagerHeader.spec.js:12]: nah, I like `daKey`.

**Review by fpigeonjr — APPROVED** [review]
looks good @jschoenemann-cms

**Pull request merged**